### PR TITLE
Term entry] UI and ux design UI ux (ideation) user pain points

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description

Term entry for UI and UX, user pain points

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Kadif-Ravik** [comment]
@dakshdeepHERE

File: content/uiux/user-pain-points/user-pain-points.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ - 'Best Practices'
+ - 'Developement'
+ - 'UI'
+ - 'UX'
+ContentCatalog:
+- 'intro-to-ui-ux'
+- 'paths/front-end-engineer-career-path'
+---
+
+**User pain points** are areas of difficulty or frustration a user encounters when using a website or program that make the experience undesirable. User pain points can be minor inconveinences to major issues preventing users from accessing the product or service entirely. Identifying and resolving these issues is a crucial part of web development and should be considered throughout any project. Pain points are commonly described in three levels with fours types at each level
```
Comment (dakshdeepHERE) [inline content/uiux/user-pain-points/user-pain-points.md:18]:
```suggestion
**User pain points** are areas of difficulty or frustration a user encounters when using a website or program that make the experience undesirable. User pain points can be minor inconveniences to major issues preventing users from accessing the product or service entirely. Identifying and resolving these issues is a crucial part of web development and should be considered throughout any project. Pain points are commonly described in three levels with four types at each level
```

File: content/uiux/user-pain-points/user-pain-points.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Pain Point Levels
+
+ - Interaction: users interaction with product or support
+ - Journey: the steps it takes a user to complete a task
+ - Relationship: users lifetime experience with a product or brand
```
Comment (dakshdeepHERE) [inline content/uiux/user-pain-points/user-pain-points.md:24]:
```suggestion
- `Interaction`: User's interaction with product or support
- `Journey`: The steps it takes a user to complete a task
- `Relationship`: User's lifetime experience with a product or brand 
```

File: content/uiux/user-pain-points/user-pain-points.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ - Interaction: users interaction with product or support
+ - Journey: the steps it takes a user to complete a task
+ - Relationship: users lifetime experience with a product or brand 
+
+## Pain Point Types
+
+ - Financial: relates to paywalls and subscription services that locks a user out.
+ - Product: relates to the performance, quality and usability issues while a using a product
+ - Process: relates to the steps required to complete a task
+ - Support: relates to how organizations respond to user questions
```
Comment (dakshdeepHERE) [inline content/uiux/user-pain-points/user-pain-points.md:31]:
```suggestion
- `Financial`: relates to paywalls and subscription services that lock a user out.
- `Product`: relates to the performance, quality, and usability issues while using a product
- `Process`: relates to the steps required to complete a task
- `Support`: relates to how organizations respond to user questions
```

File: content/uiux/user-pain-points/user-pain-points.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ 
+## Example
+
+In the example below there are many elements of the web page that are sure to cause frustration for any user. A user might find it difficult to differentiate between what is an item for sale and what is an advertisment. There are also many accessibility issues and following [Accessibility Best Practices](https://www.w3.org/WAI/tips/designing/Accessibility) will aid in solving many common issues.
```
Comment (dakshdeepHERE) [inline content/uiux/user-pain-points/user-pain-points.md:35]:
```suggestion
In the example below there are many elements of the web page that are sure to cause frustration for any user. A user might find it difficult to differentiate between what is an item for sale and what is an advertisement. There are also many accessibility issues and following [Accessibility Best Practices](https://www.w3.org/WAI/tips/designing/Accessibility) will aid in solving many common issues.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Have few requests for you to work on @Kadif-Ravik 🚀

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review @Kadif-Ravik 🚀

File: content/uiux/user-pain-points/user-pain-points.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+ - 'Best Practices'
+ - 'Developement'
+ - 'UI'
+ - 'UX'
+ContentCatalog:
+- 'intro-to-ui-ux'
+- 'paths/front-end-engineer-career-path'
+---
+
+**User pain points** are areas of difficulty or frustration a user encounters when using a website or program that make the experience undesirable. User pain points can be minor inconveinences to major issues preventing users from accessing the product or service entirely. Identifying and resolving these issues is a crucial part of web development and should be considered throughout any project. Pain points are commonly described in three levels with four types at each level
```
Comment (ishg-153) [inline content/uiux/user-pain-points/user-pain-points.md:18]:
```suggestion
**User Pain Points** are areas of difficulty or frustration for a user when using a product that makes the experience undesirable. These can be minor inconveniences or issues preventing users from accessing the product or service entirely.  User Pain Points can also be difficulties users face with their daily tasks. Such problems are identified at the beginning of the product development process, to build user-centric solutions. They can be identified through User Research within the User Experience (UX) design process.

User Pain Points within existing applications are commonly described in three levels with four types at each level given below.
```

File: content/uiux/user-pain-points/user-pain-points.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- `Financial`: relates to paywalls and subscription services that lock a user out.
+- `Product`: relates to the performance, quality, and usability issues while using a product
+- `Process`: relates to the steps required to complete a task
+- `Support`: relates to how organizations respond to user questions
+ 
+## Example
+
+In the example below there are many elements of the web page that are sure to cause frustration for any user. A user might find it difficult to differentiate between what is an item for sale and what is an advertisement. There are also many accessibility issues and following [Accessibility Best Practices](https://www.w3.org/WAI/tips/designing/Accessibility) will aid in solving many common issues.
+
+![Example of a crowded website layout](https://raw.githubusercontent.com/Codecademy/docs/main/media/user-pain-points-example.png)
```
Comment (ishg-153) [inline content/uiux/user-pain-points/user-pain-points.md:37]: This example seems a little vague. Can you use a simpler example to explain a user pain point? It isn't necessary to use an image. 

If you have trouble with this, let me know.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Kadif-Ravik I have suggested changes. Please make those.

**Comment by ishg-153** [comment]
@Kadif-Ravik It's been two weeks are you done making all the suggested changes?

**Review by ishg-153 — APPROVED** [review]
Merging this

**Pull request merged**